### PR TITLE
Gcs update

### PR DESCRIPTION
**Pull request by marcasmar1**

### CODE REVIEW

**Comment by marcasmar1** [comment]
This pull request fails the changelog test, but as the current version hasn't been uploaded, it wouldn't make any sense to increase the version.
Current version:

> # 0.6.2+schema.0.4.1 
> - Created format document guide

File: docs/src/common_uses/basic_example.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "record_number": 0,
+    "time": 1696421411207,
+    "velocity": {
+        "east": -2.14297,
+        "north": 1.69235,
+        "up": -0.86349
+    }
+},
+...
+```
```
Comment (aleokdev) [inline docs/src/common_uses/basic_example.md:49]: These lack context; I think they should be intertwined with the explanation.

**Review by aleokdev — COMMENTED** [review]
Ok, got some feedback. Also, did you check if these JSON files are successfully parsed by track2kml?

File: docs/src/common_uses/basic_example.md (hunk 1 above)
Comment (marcasmar1) [reply docs/src/common_uses/basic_example.md:49]: I don't understand what you mean with intertwined in this context.

**Comment by marcasmar1** [comment]
> Ok, got some feedback. Also, did you check if these JSON files are successfully parsed by track2kml?

Yes, I've succesfully created their corresponding kml files.

File: docs/src/common_uses/basic_example.md (hunk 1 above)
Comment (aleokdev) [reply docs/src/common_uses/basic_example.md:49]: I mean that they should appear adjacent to explanations as opposed to appearing in a separate page.

File: docs/src/track2kml/track2kml.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Visualizing data
+Data in the COURAGEOUS format can be visualized using Google Earth Pro
+using [track2kml](https://grvc.us.es/courageous/).
+
+**track2kml** is a commandline application that converts COURAGEOUS JSON files
+into KML.
+
+More information on track2kml can be found writting `track2kml --help`.
```
Comment (aleokdev) [inline docs/src/track2kml/track2kml.md]:
```suggestion
More information on track2kml can be found writing `track2kml --help`.
```

File: docs/src/track2kml/track2kml.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Executing track2kml on any of the following examples:
+- [basic COURAGEOUS example](../res/example_basic.json)
+- [intercepting C-UAS COURAGEOUS format example](../res/example_intercepting.json)
+- [additional example](../res/test_data.json) 
+
+Gives the following KML files:
```
Comment (aleokdev) [inline docs/src/track2kml/track2kml.md]:
```suggestion
Gives the following KML files (On version 0.4.2):
```

**Review by aleokdev — APPROVED** [review]

**Comment by aleokdev** [comment]
@marcasmar1 All good to merge?

**Comment by marcasmar1** [comment]
> @marcasmar1 All good to merge?

No, actually.
There's still work to do on the _Basic example_ page.

**Review by aleokdev — APPROVED** [review]
OK, looks good to me.

**Comment by aleokdev** [comment]
@marcasmar1 Can we merge now?

**Comment by marcasmar1** [comment]
> @marcasmar1 Can we merge now?

Yes

**Pull request merged**